testsuite/forge/src/main.rs

```diff
@@ -33,27 +33,79 @@ struct Args {
     )]
     duration: u64,
 
-    // operator options
-    #[structopt(long, help = "If set, wipes the state of the test backend and exits")]
-    clean_up: bool,
+    #[structopt(flatten)]
+    options: Options,
+
+    #[structopt(subcommand)]
+    ops_cmd: Option<OperatorCommand>,
+
     #[structopt(
         long,
         help = "Override the helm repo used for k8s tests",
         default_value = "testnet-internal"
     )]
     helm_repo: String,
+}
+
+#[derive(StructOpt, Debug)]
+enum OperatorCommand {
+    SetValidator(SetValidator),
+    CleanUp(CleanUp),
+}
+
+#[derive(StructOpt, Debug)]
+struct SetValidator {
+    validator_name: String,
+    #[structopt(long, help = "The image tag used for validators")]
+    image_tag: String,
+}
+
+#[derive(StructOpt, Debug)]
+struct CleanUp {
     #[structopt(long, default_value = "30")]
     num_validators: usize,
-
-    #[structopt(flatten)]
-    options: Options,
+    #[structopt(
+        long,
+        help = "Override the image tag used for validators",
+        default_value = "devnet"
+    )]
+    validator_image_tag: String,
+    #[structopt(
+        long,
+        help = "Override the image tag used for testnet-specific components",
+        default_value = "devnet"
+    )]
+    testnet_image_tag: String,
+    #[structopt(
+        long,
+        help = "If set, performs validator healthcheck and assumes k8s DNS access"
+    )]
+    require_validator_healthcheck: bool,
 }
 
 fn main() -> Result<()> {
     let args = Args::from_args();
 
-    if args.clean_up {
-        return clean_k8s_cluster(args.helm_repo, args.num_validators);
+    match args.ops_cmd {
```

Comment:
I assume we may not expose these cluster operations in Forge but maybe in a CLI tool instead. Forge test can call those operations by needed. But we can leave them here in this PR for testing purpose, the PR for compatibility test will do code cleanup here

Reply:
I think it makes sense to keep these all in Forge and not a separate CLI tool. Forge can then use the cluster operations individually via CLI or as part of a test (or test setup).

Reply:
ok, I think maybe use
```
if let Some(ops_cmd) = args.ops_cmd
```
will be cleaner

Reply:
gotcha, done